src/components/TopBar/TopBarMenu.vue

```diff
@@ -492,6 +504,24 @@ export default {
 				token: this.token,
 			})
 		},
+
+		async downloadAttendanceList() {
+			try {
+				const reponse = await callDownloadParticipants(this.token)
+				// Create a link and click it to download the file
+				const url = window.URL.createObjectURL(new Blob([reponse.data]))
+				const link = document.createElement('a')
```

Comment:
Instead of all this magic you could simply let the browser do the request directly?
The backend is setting a beautiful file name and all :see_no_evil:

Reply:
Sadly, it's not that simple
![image](https://github.com/user-attachments/assets/a228a832-2f1b-408b-9945-a446065a7df6)

Reply:
Hmm, have to add `#[Http\Attribute\NoCSRFRequired]` when it's not an API call, I see.
Also we are greeten by the "Leave call" dialog. So maybe it's better to do an intercept :see_no_evil: 

I was about to do a rename of the file thou, so maybe we just have to do it in the frontend then?
Suggestion: `<cleaned room name> - <cleaned date and time>.csv`
cleaned means: replace all "breaking" characters with `-` and replace multiple `--` with a single one.

E.g. `Hello/World:://|?Tour🌎.pdf` room would generate something like: `Hello-World-Tour-pdf - 2024-10-14 18-05.csv` ( https://github.com/nextcloud/server/blob/0936af996013ab07fe845fb852b80b3ab1d536f0/apps/files/lib/Service/SettingsService.php#L27-L31 and `/` and `\` and `.`) and limit to reasonable length, so that we don't even remotely reach the 255 character limit of Windows :see_no_evil:

Reply:
`#[Http\Attribute\NoCSRFRequired]` is enough to use NcActionLink =)
But we still need to generate a file name on a client. Don't have any cleanup util on a client side though

Reply:
> But we still need to generate a file name on a client.

With NcActionLink it worked for me without? The server is setting the correct Header and therefore it downloads as `participant.csv` for me atm.